pkg/kubelet/status/status_manager.go

```diff
@@ -151,7 +162,7 @@ func NewManager(kubeClient clientset.Interface, podManager PodManager, podDeleti
 		kubeClient:              kubeClient,
 		podManager:              podManager,
 		podStatuses:             make(map[types.UID]versionedPodStatus),
-		podResizeStatuses:       make(map[types.UID]v1.PodResizeStatus),
+		podResizeStatuses:       make(map[types.UID]PodResizeStatus),
```

Comment:
Need to delete entries in `deletePodStatus` and `RemoveOrphanedStatuses`

Reply:
maybe I'm missing something but `deletePodStatus` and `RemoveOrphanedStatuses` are already deleting the entry?

Reply:
Right, sorry. I forgot you were reusing the same map. It's more obvious now with the rename. :+1: